Makefile

```diff
@@ -8,8 +8,11 @@ cert-tool: $(CERT_TOOL_FILES)
 .PHONY: certs
 certs: cert-tool
 	rm -rf certs && mkdir certs
-	cert-tool -CA
-	cert-tool -cert-out certs/server.cert -key-out certs/server.key -dc-capable server
+	./cert-tool -CA
+	./cert-tool -cert-out certs/server.cert -key-out certs/server.key -dc-capable server
 
 dc: certs/server.cert certs/server.key cert-tool
-	cert-tool -cert-in certs/server.cert -key-in certs/server.key -generate-dc -dc-out certs/dc.txt
+	./cert-tool -cert-in certs/server.cert -key-in certs/server.key -generate-dc -dc-out certs/dc.txt
+
+clean:
+	docker system prune -a
```

Comment:
Could you replace this with [docker builder prune](https://docs.docker.com/engine/reference/commandline/builder_prune/)? Just found out about this option.